employee-rostering-frontend/src/store/tenant/operations.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Tenant from 'domain/Tenant';
+import { ThunkCommandFactory } from '../types';
+import * as actions from './actions';
+import { ChangeTenantAction, RefreshTenantListAction } from './types';
+import {refreshSkillList} from 'store/skill/operations';
+
+export const changeTenant: ThunkCommandFactory<number, ChangeTenantAction> = tenantId =>
+  (dispatch, state, client) => {
+    dispatch(actions.changeTenant(tenantId));
+    refreshSkillList();
```

Comment:
You can dispatch other actions/operations here but not like this. You can't call the operation directly. You literally need to **dispach** it like this:
```ts
dispatch(refreshSkillList());
```